src/components/pages/LandingPage.vue

```diff
@@ -1,24 +1,41 @@
 <script>
-    import { defineComponent } from 'vue';
+import { defineComponent } from 'vue';
 import TheHeader from '@/components/views/TheHeader.vue';
-import TopNavbar from '@/components/features/TopNavbar.vue';
-import SignSection from '@/components/features/SignSection.vue';
+import TopNavbar from '@/components/features/Header/TopNavbar.vue';
+import SignSection from '@/components/features/Header/SignSection.vue';
 import TheContainer from '@/components/common/TheContainer.vue';
 import WelcomeSection from '@/components/views/WelcomeSection.vue';
-import MovieCard from '@/components/views/MovieCard.vue';
 import MoviesSummary from '@/components/views/MoviesSummary.vue';
+import { getAllMovies } from '@/api/service/movies';
 import ScreeningsFilters from '@/components/views/ScreeningsFilters.vue';
+
 export default defineComponent({
   components: {
     TheHeader,
     TopNavbar,
     SignSection,
     TheContainer,
     WelcomeSection,
-    MovieCard,
     MoviesSummary,
     ScreeningsFilters
-}
+  },
+  data() {
+    return {
+      movies: [],
+      filteredMovies: [],
+    };
+  },
+  async mounted() {
+    try {
+      const response = await getAllMovies();
+      this.movies = response.data;
+    } catch {
+      this.$router.push({ name: 'NoMatch' });
```

Comment:
Do I understand properly? If there is no movies, you are redirecting user to another page?  What idea is behind that? 🧐

Reply:
"NoMatch" is my error-message-page. I can use console.error(error) instead, but then user wouldn't know that something went wrong. But maybe he/she/it doesn't have to? 😄

Reply:
I think more expected behavior will be just hiding away from user this section. User still will be able to explore homepage. Right now it can be a bit confusing 🙂 

tip => I think you can change file name to `Homepage` as it is a homepage. I feel like landing page is more specific to design world😅

Reply:
Ok! The section will not appear without movies so I will go for displaying error in the console only.
And, yes, HomePage sounds much better 😸